hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -2475,4 +2475,72 @@
       hops.ug.cache.secs to 0.
     </description>
   </property>
+
+  <property>
+    <name>hops.crl.validation.enabled</name>
+    <value>false</value>
+    <description>Enable CRL validation when HopsTLS is enabled</description>
+  </property>
+
+  <property>
+    <name>hops.crl.fetcher.class</name>
+    <value>org.apache.hadoop.security.ssl.RemoteCRLFetcher</value>
+    <description>Implementation class to use to fetch the Certificate Revocation List</description>
+  </property>
+
+  <property>
+    <name>hops.crl.input.uri</name>
+    <value>https://host:port/path/to/crl</value>
+    <description>URI to download CA CRL</description>
+  </property>
+
+  <property>
+    <name>hops.crl.output.file</name>
+    <value>/tmp/crl.pem</value>
+    <description>File path to save CRL</description>
+  </property>
+
+  <property>
+    <name>hops.crl.fetcher.interval</name>
+    <value>720</value>
+    <description>
+      Interval used by RevocationListFetcherService to fetch CA's CRL.
+      Valid suffixes NS, US, MS, S, M, H, D
```

Comment:
Is no suffix also a valide suffix? otherwise why is there no sufix for the value?

Reply:
You're correct. I will update the description